### PR TITLE
fix(client): OS specific submit button text

### LINKED ISSUE #55627: Submit exercise shortcut
### Is your feature request related to a problem? Please describe.

Freecodecamp Curriculum

### Describe the solution you'd like

At the moment some basic shortcuts exist on the platfrom, one of those shortcuts being `ctrl` + `enter` which checks the code, but after checking the code you would have to click the submit button manually, maybe it would be a good idea to add the same `ctrl` + `enter` shortcut to the submit button.
image
image

### Describe alternatives you've considered

Another idea would be to have a live code check and one button which submits the code.

### Additional context

There is another shortcut: `esc` + `n` that navigates to the next exercise,  which we can access by enabling the extra shortcuts from settings, but this shortcut only navigates and doesn't submit the exercise.

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #55627

### CODE REVIEW

**Comment by a2937** [comment]
This does look pretty good, although I think a test should be added to ensure that the correct words are displaying depending on the browser type.

**Comment by gagan-bhullar-tech** [comment]
@a2937 added test case

**Review by lasjorg — CHANGES_REQUESTED** [review]
I would prefer if your code used the same syntax as the other button code in the file. It is just less confusing to read if they are both written the same way.

**Review by lasjorg — COMMENTED** [review]
@gagan-bhullar-tech I forgot to mention it, but can you also move it up, so the code for the two buttons are one after the other.

So before

`const showSignInButton = !isSignedIn && challengeIsCompleted;`

File: client/src/templates/Challenges/classic/lower-jaw.tsx [hunk 1]
```diff
@@ -293,7 +293,14 @@ const LowerJaw = ({
       : t('buttons.check-code')
     : t('buttons.check-code-2');
 
+  const submitButtonText = isDesktop
+    ? isMacOS
+      ? t('buttons.submit-and-go-3')
+      : t('buttons.submit-and-go')
+    : t('buttons.submit-and-go-2');
```
Comment (lasjorg) [inline client/src/templates/Challenges/classic/lower-jaw.tsx:300]:
```suggestion
      : t('buttons.submit-and-go-2')
    : t('buttons.submit-and-go');
```

Sorry, the properties are apparently switched for the submit button text.

I might open a PR after this is merged to give them better names and make sure the default is consistent.

**Review by lasjorg — CHANGES_REQUESTED** [review]

**Review by lasjorg — APPROVED** [review]
LGTM.

I can't test it on Mac, but I see no reason why it wouldn't work.

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**